.github/workflows/comment.yaml

```diff
@@ -38,14 +38,15 @@ jobs:
         parts=(${REPO_FULL_NAME//\// })
         echo "ORG=${parts[0]}" >> $GITHUB_OUTPUT
         echo "REPO=${parts[1]}" >> $GITHUB_OUTPUT
+        echo "REPO_FULL_NAME=${REPO_FULL_NAME}" >> $GITHUB_OUTPUT
         echo "SHA=${{ github.event.pull_request.head.sha }}" >> $GITHUB_OUTPUT
         echo "IS_PR=${{ github.event_name == 'pull_request' }}" >> $GITHUB_OUTPUT
     - name: Workflow run setup
       id: run
       if: github.event_name == 'workflow_run'
       run: |
         echo "RUN_ID=${{ github.event.workflow_run.id }}" >> $GITHUB_OUTPUT
-        echo "ORG=${{ github.event.workflow_run.organization.login }}" >> $GITHUB_OUTPUT
+        echo "ORG=${{ github.event.workflow_run.repository.owner.login }}" >> $GITHUB_OUTPUT
```

Comment:
Does the workflow still work correctly if the repository owner is not an organization?
Would it be better to rename the `ORG` to `OWNER`?

Reply:
good catch, lets' go!  thanks @yhaliaw